cordova-js-src/exec.js

```diff
@@ -60,18 +60,32 @@ module.exports = function (success, fail, service, action, args) {
             // CB-5806 [Windows8] Add keepCallback support to proxy
             onSuccess = function (result, callbackOptions) {
                 callbackOptions = callbackOptions || {};
+                var callbackStatus;
+                if (callbackOptions.status !== null) {
```

Comment:
Yes it is - in the event that the user passes a callbackOptions but does not specify status, it will return null.